### ISSUE TITLE
Label doesn't resize when text changes in StackLayout

### ISSUE THREAD

**Issue by trampster** [picoe/Eto#1467]
## Expected Behavior
Label should resize when the text changes, so that it fits the new text. 

## Actual Behavior
Label text is clipped. In the code below when the MainViewModel.HomeGroupName changes and the new text is wider then the original then it gets clipped, rather then resizing:

For example Changing from 'Hydra' to 'PocOnly' results in the following:

Before:
![image](https://user-images.githubusercontent.com/2063633/63062331-96353580-bf4c-11e9-8729-9668e4ce7b65.png)

After (clipped):

![image](https://user-images.githubusercontent.com/2063633/63062346-ab11c900-bf4c-11e9-81f5-cb76c9e5b13c.png)

## Steps to Reproduce the Problem

  1. Use the code below
  2. Change the text bound to groupLabel to be larger
  3. Notice it is now clipped instead of resizing.

## Code that Demonstrates the Problem

```
	var groupLabel = new Label();
	groupLabel.Font = new Font(FontFamilies.Sans, 24);
	groupLabel.BindDataContext(c => c.Text, (MainViewModel model) => model.HomeGroupName);

	var stack = new StackLayout
	{
		Orientation = Orientation.Horizontal,
		VerticalContentAlignment = VerticalAlignment.Center,
	};

	var downImage = _icons.ChevronDown;

	stack.Items.Add(new StackLayoutItem(groupLabel, true));
	stack.Items.Add(downImage);

	var verticalStack = new StackLayout()
	{
		Orientation = Orientation.Vertical,
		HorizontalContentAlignment = HorizontalAlignment.Center,
	};
	verticalStack.Items.Add(new StackLayoutItem(stack, true));

```

## Specifications

  - Version: 2.4.1
  - Platform(s): Gtk3
  - Operating System(s): Ubuntu 18.04

**Comment by trampster** [comment]
It is important to note that I want the text+image to be centred in the form, so setting the label to stretch will not do what I want.

**Comment by trampster** [comment]
I'm open to any alternative layout method that will achieve my desired layout.

**Comment by cwensley** [comment]
Hey @trampster, have you tried 2.5?  I think this has been fixed.

**Comment by trampster** [comment]
Yes I can confirm this is fixed in 2.5-rc.3
